scripts/jenkins/jobs-ibs/cloud-mkcloud6-gating.yaml

```diff
@@ -36,11 +37,21 @@
             - name: openstack-mkcloud
               node-label: cloud-trigger
               predefined-parameters: |
+                TESTHEAD=1
                 cloudsource=develcloud6
                 nodenumber=4
                 want_ldap=1
                 networkingplugin=linuxbridge
                 mkcloudtarget=all
+            - name: openstack-mkcloud
```

Comment:
we could try to make this
name: cloud-mkcloud6-job-ha
and drop all predefined-parameters below.